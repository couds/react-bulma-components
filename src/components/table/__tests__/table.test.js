import React from 'react';
import { render } from 'react-testing-library';
import { Table } from '../Table';

describe('Table component', () => {
  it('should render', () => {
    const { asFragment } = render(<Table />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should concat classname in props with classname', () => {
    const { asFragment } = render(
      <Table className="other-class this-is-a-test">
        <p>Default</p>
      </Table>
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('should use inline styles', () => {
    const { asFragment } = render(
      <Table style={{ height: 250 }}>
        <p>Default</p>
      </Table>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
