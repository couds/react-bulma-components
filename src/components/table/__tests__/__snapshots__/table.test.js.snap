// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component should concat classname in props with classname 1`] = `
<DocumentFragment>
  <p>
    Default
  </p>
  <table
    class="table other-class this-is-a-test is-fullwidth is-striped"
  />
</DocumentFragment>
`;

exports[`Table component should render 1`] = `
<DocumentFragment>
  <table
    class="table is-fullwidth is-striped"
  />
</DocumentFragment>
`;

exports[`Table component should use inline styles 1`] = `
<DocumentFragment>
  <p>
    Default
  </p>
  <table
    class="table is-fullwidth is-striped"
    style="height: 250px;"
  />
</DocumentFragment>
`;
